README.md

```diff
@@ -24,6 +24,140 @@ Rules
+
+If you came to the conclusion that you require a new parameter defintion, copy the default block from above, or simply any other parameter in the list. Don't forget to update all fields to match the purpose and type of your parameter.
+
+### **Updating value translations**
+
+The main purpose of this project is to be able to map parameters across tools. Some tools also require a value translation to properly work with the provided parameter. A prominent example are boolean values, which have to be translated to 0/1 or even tool specific flags.
+
+To add/update a value translation you have to add a list of translations into the value_translations dict:
+
+```
+{
+        "default_value": false,
+        "description": "Reduces the number of spurious junctions",
+        "key_translations": {
+            "star_style_1": "--outFilterType",
+            "ursgal_style_1": "aligner_filter_spurious_junctions"
+        },
+        "name": "aligner_filter_spurious_junctions",
+        "tag": [
+            "alignment"
+        ],
+        "triggers_rerun": true,
+        "value_translations": {
+            "star_style_1": [
+                [
+                    false,
+                    "Normal"
+                ],
+                [
+                    true,
+                    "BySJout"
+                ]
+            ],
+            "ursgal_style_1": [
+                [
+                    false,
+                    false
+                ],
+                [
+                    true,
+                    true
+                ]
+            ]
+        },
+        "value_type": "bool"
+    }
+```
+
+the value translations are a list of lists for your desired style. Each individual value translation is a list of two elements: `[ursgal_param_value, tool_param_value]`.
+**Note:** to make the whole translation logic work in any direction, the value translations **MUST** also be provided for `ursgal_style_1`. As you can see in the example, the value is simply mapped to itself. 
+
+
+### **Special cases**
+
+Some key_translations values cannot be directly mapped to a tool specific parameter. This is due to the harmonization of parameters in this project and different parameter handling of the tools. For example, if considereing tolerance parameters some tools accept symmetric windows, while others can handle asymmetric windows. The representation of such can be either one parameter, where the tool internally applies it in both direction from the measured value (only for symmetric windows), or in turn a list of lower and upper boundary. 
+
+These parameters require further tool specific processing, prior being injected e.g. into a command line. You can easily identify them, as they have a **\<tag\>** in their names. In the following example:
+```
+"cutadapt_style_1": "-u_<part_read_start>",
```

Comment:
is 1 missing in part1?

Reply:
@tristan-ranff will it break something in cut adapt if I update the param name? Not sure if you are calling the full tag in your wrapper